src/customfunctions.ts

```diff
@@ -0,0 +1,25 @@
+﻿declare var CustomFunctionMappings;
+
+function add(first: number, second: number): number {
+  return first + second;
+}
+
+function addAsync(first: number, second: number): Promise<number> {
+  return Promise.resolve(add(first, second));
+}
+
+function increment(incrementBy: number, callback) {
+  let result = 0;
+  const timer = setInterval(() => {
+    result += incrementBy;
+    callback.setResult(result);
+  }, 1000);
+
+  callback.onCanceled = () => {
+    clearInterval(timer);
+  };
+}
+
+CustomFunctionMappings.add = add;
```

Comment:
Since CustomFunctionMapping is so new, can we add a comment around these to explain and give context?

Reply:
Also I am not sure but I think it is CustomFunctionMapping with no S at the end :)

Reply:
@mscharlock You make a good point that we need to provide better explanation, either in the code or in README documentation. We're going to continue to make updates to this, and I don't think it is required as part of this pull request.